### ISSUE TITLE
test_tutorial_coherence seems to take much longer than before

### ISSUE THREAD

**Issue by robertoostenveld** [fieldtrip/fieldtrip#1903]
As of 13 October the `test_tutorial_coherence.m` and `test_tutorial_connectivity20130308 .m` scripts take too long to complete on the DCCN compute cluster and get killed. 

```
PBS Job Id: 44480921.dccn-l029.dccn.nl
Job Name:   test_tutorial_coherence
Exec host:  dccn-c036.dccn.nl/17
Aborted by PBS Server 
Job exceeded its walltime limit. Job was aborted
See Administrator for help
Exit_status=-11
resources_used.cput=00:19:51
resources_used.vmem=7498624kb
resources_used.walltime=00:20:37
resources_used.mem=2199988kb
resources_used.energy_used=0
req_information.task_count.0=1
req_information.lprocs.0=1
req_information.total_memory.0=5242880kb
req_information.memory.0=5242880kb
req_information.thread_usage_policy.0=allowthreads
req_information.hostlist.0=dccn-c036.dccn.nl:ppn=1
req_information.task_usage.0.task.0={"task":{"cpu_list":"7","mem_list":"1","cores":0,"threads":1,"host":"dccn-c036.dccn.nl"}}
Error_Path: mentat002.dccn.nl:/home/megmethods/roboos/fieldtrip/dashboard/logs/144f3d3/test_tutorial_coherence.err
Output_Path: mentat002.dccn.nl:/home/megmethods/roboos/fieldtrip/dashboard/logs/144f3d3/test_tutorial_coherence.txt
```

Since both make use of ft_connectivityanalysis, I wonder whether something in that function (possibly simply bookkeeping) was made slower in a recent commit.

**Comment by schoffelen** [comment]
OK, I think that I traced this one down: read_ctf_ascii is used to read a headmodel file in ctf's .hdm format. The text reading has become extremely slow, because for each line that is read, ft_warning is called twice, once switching all warnings off, and once switching to the previous state. The slowdown seems to be caused by some person (hehe) introducing gettimestamp/settimestamp functionality for ft_notification, in 

https://github.com/fieldtrip/fieldtrip/commit/4be70887309d225044bf1dc790e3321dc8325e92#diff-ce8054d18e01a416f80f51f5d5914e0fa2b1563168124b8c8cea073f413177fe

I changed the multiple calls to ft_warning in read_ctf_ascii for now, and the reported 'slow' test code now runs through fine in terms of execution time. 

Should we profile the ft_notification a bit more before closing this? @robertoostenveld
